scss/_patterns_tabs.scss

```diff
@@ -71,20 +66,76 @@
         }
       }
 
+      &:active,
+      &[aria-selected='true'] {
+        // Hide the highlight when focussing (in combination with the parent
+        // states) in browsers that don't support focus-visible.
+        &:focus::before,
+        &:focus::after {
+          content: none;
+        }
+      }
+    }
+  }
+  
+  // Theming
+  @if ($theme-default-p-tabs == 'dark') {
+    .p-tabs {
+      @include vf-p-tabs-dark-theme;
+    }
+
+    .p-tabs.on-light {
+      @include vf-p-tabs-light-theme;
+    }
+  } @else {
+    .p-tabs {
+      @include vf-p-tabs-light-theme;
+    }
+
+    .p-tabs.on-dark {
+      @include vf-p-tabs-dark-theme;
+    }
+  }
+}
+
+
+@mixin vf-p-tabs-theme($colors--text-default, $colors--border-default) {
+  color: $colors--text-default;
+.p-tabs {
+    &__list::after {
+      background-color: $colors--border-default;
+    }
+
+    &__link {
+      @extend %vf-button-base;
+      @include vf-button-pattern(
```

Comment:
Do we even need that if it's all transparent? What's the purpose of that?

Reply:
well spotted, copy paste mistake. removed

Reply:
because only the disabled background is transparent by default - the others apply light theme colours. So we have to specify we want a behaviour different form the default